dbms/src/Storages/DeltaMerge/PKSquashingBlockInputStream.h

```diff
@@ -42,6 +43,15 @@ class PKSquashingBlockInputStream final : public IBlockInputStream
         children.push_back(child);
     }
 
+    ~PKSquashingBlockInputStream() override
```

Comment:
Aims to find out if there are too many rows in one block, thus cost too much time sorting